package-scripts.js

```diff
@@ -86,7 +86,7 @@ module.exports = {
       'clean': 'if exist dist rmdir dist /s /q',
       'compile': 'tsc -p server/tsconfig.json',
       'copy-assets': 'copy server\\package.json dist\\server\\package.json && (robocopy server\\assets dist\\server\\assets /e || echo 0)',
-      'install-node-modules': 'cd dist\\electron && yarn',
+      'install-node-modules': 'cd dist\\server && yarn',
```

Comment:
Just making sure, was this intentional? It seems unrelated

Reply:
Yup. It's intentional.